e107_plugins/hero/templates/hero_template.php

```diff
@@ -53,7 +53,7 @@
  											</div>
 					                      <div class="hero-buttons text-right">
 					     
-					                          <a href="{HERO_BUTTON1_URL}" class="btn btn-{HERO_BUTTON1_CLASS} btn-raised animated fadeInRight animation-delay-28">
+					                          <a target="_blank" href="{HERO_BUTTON1_URL}" class="btn btn-{HERO_BUTTON1_CLASS} btn-raised animated fadeInRight animation-delay-28">
```

Comment:
@yesszus Why this change?

Reply:
Content that points to an external link do not leave the website.

Reply:
@yesszus Thanks. But we do not know for sure that it is an external link.

Reply:
Ok.